examples/simple/simple.tf

```diff
@@ -1,5 +1,5 @@
 provider "aws" {
-  version = "~> 3.0"
+  version = "~> 4.0"
```

Comment:
(this comment is not for this line)
Don't forget to update the README and examples to use a new version of this terraform module. My guess is to upgrade to v7.0.0?